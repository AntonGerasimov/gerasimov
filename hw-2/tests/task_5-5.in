23 21 15
2.3 3.2 5.5
