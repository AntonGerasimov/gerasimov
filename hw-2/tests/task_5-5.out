59
11.000000
